Medicare/src/DatabaseAccessLayer/DAO.java

```diff
@@ -71,420 +74,238 @@ public ResultSet getRs() {
     public void setRs(ResultSet rs) {
         this.rs = rs;
     }
-    
-     public void DoConnect( ) {
 
-       try {
+    public String getLoggedInUser() {
+        return loggedInUser;
+    }
+
+    public void setLoggedInUser(String loggedInUser) {
+        this.loggedInUser = loggedInUser;
+    }
+
+    public void DoConnect() {
+
+        try {
             InputStream bdl = this.getClass().getResourceAsStream("/resources/dbconfig.properties");
             Properties prop = new Properties();
             prop.load(bdl);
             String host = "jdbc:mysql://" + prop.getProperty("host") + "/" + prop.getProperty("dbname");

```

Comment:
define all of these as constants on the top of the file:
jdbc:mysql
host
dbname
user
password